schemas/irc-request.yml

```diff
@@ -12,6 +12,7 @@ properties:
       either `user` or `channel`, you cannot supply both.
   user:
     type:           string
+    pattern:        "^[A-Za-z\\[\\]\\\\~_\\^{|}][A-Za-z0-9\\-\\[\\]\\\\~_\\^{|}]{0,254}$",
```

Comment:
note on irc.mozilla.org, it looks like there is a 31 char limit - so we might want to change this to `{0,30}` to be safe and update `maxLength` to 31 - of course that limit may be higher on other IRC servers, so we might not want to...